paddle/fluid/eager/auto_code_generator/final_state_generator/eager_gen.py

```diff
@@ -40,8 +40,10 @@
 # keeping the code compatible, here we also skip inplace check in new dygraph temporarily,
 # and this will be fixed in the futrue.
 inplace_check_blacklist = set(["assign_out_"])
-# # --- Black Ops list that's NO NEED to apply backward code generation
-black_ops_list = ["conv2d", "conv2d_grad", "conv2d_grad_grad", "add_n"]
+
+# Black Ops list that's NO NEED to apply forward and backward code generation,
+# for example, "conv2d" in black_ops_list, means that conv2d, conv2d_grad and conv2d_grad_grad NO NEED to apply code generation
+black_ops_list = ["conv2d", "add_n"]
```

Comment:
Make ops to apis, so that we can simply generate fwd api, bwd api, double bwd api and so on.